crates/apollo-compiler/test_data/diagnostics/0059_extension_orphan.graphql

```diff
@@ -0,0 +1,19 @@
+type Query {
```

Comment:
sorry, you're going to have to renumber these 😭

Reply:
:D